packages/yourloops/test/pages/stepper/signup-account-selector.test.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { expect } from "chai";
+import { mount, ReactWrapper } from "enzyme";
+import _ from "lodash";
+
+import { SignUpFormStateProvider } from "../../../pages/signup/signup-formstate-context";
+import SignupAccountSelector from "../../../pages/signup/signup-account-selector";
+import { UserRoles } from "../../../models/shoreline";
+
+function TestSignupAccountForm(): void {
+
+  const signupForm = (): JSX.Element => {
+    return (
+      <SignUpFormStateProvider>
+        <SignupAccountSelector handleBack={_.noop} handleNext={_.noop} />
+      </SignUpFormStateProvider>);
+  };
+
+  const nextButtonDisabled = (wrapper: ReactWrapper) => {
+    expect(wrapper.find("button#button-signup-steppers-next").prop("disabled")).to.be.true;
+  };
+
+  const nextButtonEnabled = (wrapper: ReactWrapper) => {
+    expect(wrapper.find("button#button-signup-steppers-next").prop("disabled")).to.be.false;
+  };
+
+  it("should disable next button when nothing is selected", async () => {
+    const wrapper = await mount(signupForm());
```

Comment:
opt: Might be possible to into a beforeEach: 
`move const wrapper = await mount(signupForm());`
and in an afterEach:
`wrapper.unmount();`
But it is really not necessary and might not be compatible with future tests :man_teacher:

Reply:
For info:
The unmount() is necessary if you load your component to a DOM element attached to the document.
like:
```javascript
const div = document.createElement("div");
document.body.appendElement(div);
const wrapper = await mount(signupForm(), div);
```

Reply:
done